robotiq_ethercat/CMakeLists.txt

```diff
@@ -2,19 +2,22 @@ cmake_minimum_required(VERSION 2.8.3)
 project(robotiq_ethercat)
 
 find_package(catkin REQUIRED COMPONENTS
-  ethercat_soem
+  soem
   roscpp
 )
 
 catkin_package(
    INCLUDE_DIRS include
    LIBRARIES robotiq_ethercat
-   CATKIN_DEPENDS ethercat_soem roscpp
+   CATKIN_DEPENDS soem roscpp
 )
 
-include_directories(include)
-include_directories(
+include_directories(include
   ${catkin_INCLUDE_DIRS}
+  ## The following work around allows SOEM headers to include other SOEM headers.
+  ## SOEM headers assume all headers are installed in a flat directory structure
+  ## See https://github.com/smits/soem/issues/4 for more information.
+  ${soem_INCLUDE_DIRS}/soem
```

Comment:
I'm confused: does this variable (`soem_INCLUDE_DIRS`) exist? I'd always understood that because you list `soem` as a `COMPONENT` in `find_package(catkin ..)` that `catkin_INCLUDE_DIRS` collects all include paths, and that no variables for all individual pkgs are created.